config/locales/en/errors.yml

```diff
@@ -425,6 +425,10 @@ en:
               inclusion: Select yes if your client’s outgoings are more than their income
             how_manage:
               blank: Enter how your client manages if their outgoings are more than their income
+        steps/capital/add_saving_form:
+          attributes:
+            saving_type:
+              blank: Select which savings your client has inside or outside the UK, or select 'My client does not have any of these savings'
```

Comment:
@hiboabd As per the sheet blank error message should be "Select which savings your client owns or part-owns inside or outside the UK, or select 'My client does not own any of these assets'"

Also there is typo in the google sheet. instead of "assets" in the above statement, it should be "savings"

Reply:
Yes thanks there is a typo, the owns is wrong too as the savings question is `Which savings does your client have inside or outside the UK`. So my error message should be correct as owns is for the assets question. I'll amend the spreadsheet